pkg/exchange/coinbase/api/v1/get_orders_request.go

```diff
@@ -0,0 +1,89 @@
+package coinbase
+
+import (
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/c9s/requestgen"
+)
+
+type OrderStatus string
+
+const (
+	OrderStatusOpen     OrderStatus = "open"
+	OrderStatusPending  OrderStatus = "pending"
+	OrderStatusRejected OrderStatus = "rejected"
+	OrderStatusDone     OrderStatus = "done"
+	OrderStatusActive   OrderStatus = "active"
+	OrderStatusReceived OrderStatus = "received"
+	OrderStatusAll      OrderStatus = "all"
+)
+
+func (s *OrderStatus) GlobalOrderStatus() types.OrderStatus {
+	switch *s {
+	case OrderStatusRejected:
+		return types.OrderStatusRejected
+	case OrderStatusOpen, OrderStatusPending, OrderStatusDone, OrderStatusActive, OrderStatusReceived, OrderStatusAll:
+		return types.OrderStatus(strings.ToUpper(string(*s)))
+	}
+	return types.OrderStatus(strings.ToUpper(string(*s)))
+}
+
+type Order struct {
+	Type      string      `json:"type" db:"type"`
+	Size      json.Number `json:"size" db:"size"`
+	Side      string      `json:"side" db:"side"`
+	ProductID string      `json:"product_id" db:"product_id"`
```

Comment:
if it's an integer in string format, you can use strint.Int64

Reply:
For example, the product id is `BTC-USD`.
It's not a integer in string format.